src/video.test.ts

```diff
@@ -0,0 +1,128 @@
+import { Atoms } from '@guardian/content-api-models/v1/atoms';
+import { BlockElement } from '@guardian/content-api-models/v1/blockElement';
+import { ElementType } from '@guardian/content-api-models/v1/elementType';
+import { Atom } from '@guardian/content-atom-model/atom';
+import { AtomData } from '@guardian/content-atom-model/atomData';
+import { AtomType } from '@guardian/content-atom-model/atomType';
+import { ContentChangeDetails } from '@guardian/content-atom-model/contentChangeDetails';
+import { DisplayType } from '@guardian/content-atom-model/explainer/displayType';
+import { AssetType } from '@guardian/content-atom-model/media/assetType';
+import { Category } from '@guardian/content-atom-model/media/category';
+import { MediaAtom } from '@guardian/content-atom-model/media/mediaAtom';
+import { Platform } from '@guardian/content-atom-model/media/platform';
+import { none, some } from '@guardian/types';
+import { Int64 } from 'thrift';
+import { parseVideo } from 'video';
+
+describe('parseVideo', () => {
+	let blockElement: BlockElement;
+	let atoms: Atoms;
+	let atom: Atom;
+	let atomData: AtomData;
+	let mediaAtom: MediaAtom;
+
+	beforeEach(() => {
+		blockElement = {
+			type: ElementType.VIDEO,
+			assets: [
+				{
+					type: AssetType.VIDEO,
+				},
+			],
+			contentAtomTypeData: {
+				atomId: 'atomId2',
+				atomType: 'someAtomType',
+			},
+		};
+
+		mediaAtom = {
+			assets: [
+				{
+					assetType: AssetType.VIDEO,
+					id: 'asset-id',
+					version: new Int64(2),
+					platform: Platform.YOUTUBE,
+				},
+			],
+			activeVersion: new Int64(2),
+			title: 'mediaTitle',
+			category: Category.DOCUMENTARY,
+			posterUrl: 'poster-url',
+		};
+
+		atomData = {
+			kind: 'media',
+			media: mediaAtom,
+		};
+
+		atom = {
+			id: 'atomId2',
+			atomType: AtomType.MEDIA,
+			labels: ['someLabel'],
+			defaultHtml: 'someHtml',
+			data: atomData,
+			contentChangeDetails: {} as ContentChangeDetails,
+			commissioningDesks: ['someDesk'],
+		};
+
+		atoms = {
+			media: [atom],
+		};
+	});
+
+	test('returns video', () => {
+		const expected = some({
+			posterUrl: 'poster-url',
+			videoId: 'asset-id',
+			duration: undefined,
+			atomId: 'atomId2',
+			title: 'mediaTitle',
```

Comment:
It's a nit but I'd reference these values as common variables so they live in one place?
Something like top level:
```
// in the same scope as `let blockElement: BlockElement;`
const atomId = 'atomId2'

const expected = some({
	...
	atomId,
}
```
If that makes sense?

Reply:
Yes that makes sense. Will do the change and let you know :)